stripzip.py

```diff
@@ -33,15 +43,17 @@ def purify_extra_data(mm, offset, length):
 
             if header_id in (EXTENDED_TIME_DATA, UNIX_EXTRA_DATA):
                 values[0] = STRIPZIP_OPTION_HEADER
-                for i in xrange(2, len(values)):
```

Comment:
I guess the tests for this tool in travis are just smoke tests, otherwise this should have thrown up that it wasn't python3 compatible - sounds like an issue

Reply:
true :S